src/utils/branded-types.ts

```diff
@@ -0,0 +1,15 @@
+declare const __brand: unique symbol;
```

Comment:
Nice!  

Could we move these into `src/types.ts`

Reply:
All done 🙏🏾